src/openzaak/components/zaken/openapi.yaml

```diff
@@ -20625,6 +20700,397 @@ components:
+            worden dan wel overgebracht moet worden naar een archiefbewaarplaats.
+            Wordt automatisch berekend bij het aanmaken of wijzigen van een RESULTAAT
+            aan deze ZAAK indien nog leeg.
+        resultaat:
+          type: string
+          format: uri
+          readOnly: true
+          nullable: true
+          description: URL-referentie naar het RESULTAAT. Indien geen resultaat bekend
+            is, dan is de waarde 'null'
+        opdrachtgevendeOrganisatie:
+          type: string
+          readOnly: true
+          description: De krachtens publiekrecht ingestelde rechtspersoon dan wel
+            ander niet-natuurlijk persoon waarbinnen het (bestuurs)orgaan zetelt dat
+            opdracht heeft gegeven om taken uit te voeren waaraan de zaak invulling
+            geeft.
+          maxLength: 9
+        processobjectaard:
+          type: string
+          readOnly: true
+          title: Procesobjectaard
+          description: Omschrijving van het object, subject of gebeurtenis waarop,
+            vanuit archiveringsoptiek, de zaak betrekking heeft.
+          maxLength: 200
+        startdatumBewaartermijn:
+          type: string
+          format: date
+          readOnly: true
+          nullable: true
+          description: De datum die de start markeert van de termijn waarop het zaakdossier
+            vernietigd moet worden.
+        processobject:
+          allOf:
+          - $ref: '#/components/schemas/Processobject'
+          readOnly: true
+          nullable: true
+          description: Specificatie van de attribuutsoort van het object, subject
+            of gebeurtenis  waarop, vanuit archiveringsoptiek, de zaak betrekking
+            heeft en dat bepalend is voor de start van de archiefactietermijn.
+      required:
+      - archiefactiedatum
+      - archiefnominatie
+      - archiefstatus
+      - betalingsindicatie
+      - betalingsindicatieWeergave
+      - bronorganisatie
+      - communicatiekanaal
+      - communicatiekanaalNaam
+      - deelzaken
+      - eigenschappen
+      - einddatum
+      - einddatumGepland
+      - hoofdzaak
+      - identificatie
+      - kenmerken
+      - laatsteBetaaldatum
+      - omschrijving
+      - opdrachtgevendeOrganisatie
+      - opschorting
+      - processobject
+      - processobjectaard
+      - productenOfDiensten
+      - publicatiedatum
+      - registratiedatum
+      - relevanteAndereZaken
+      - resultaat
+      - rollen
+      - selectielijstklasse
+      - startdatum
+      - startdatumBewaartermijn
+      - status
+      - toelichting
+      - uiterlijkeEinddatumAfdoening
+      - url
+      - uuid
+      - verantwoordelijkeOrganisatie
+      - verlenging
+      - vertrouwelijkheidaanduiding
+      - zaakgeometrie
+      - zaakinformatieobjecten
+      - zaakobjecten
+      - zaaktype
+    ZaakVerlengingSubRequest:
+      type: object
+      properties:
+        opschorting:
```

Comment:
Not sure why, but this includes `opschorting` instead of `verlenging`

<img width="835" height="705" alt="afbeelding" src="https://github.com/user-attachments/assets/09554ae9-82f5-46b5-86e0-4cb98a3ad59d" />